src/test/resources/com/commercetools/sync/products/helpers/variantReferenceResolver/attributes/nested-attribute-with-set-of-product-reference-attributes.json

```diff
@@ -0,0 +1,46 @@
+{
+  "key": "key",
+  "sku": "sku",
+  "attributes": [
+    {
+      "name": "attribute-name",
+      "value": [
+        {
+          "value": [
+            {
+              "id": "39611a73-9b45-414b-be70-2a207386678c",
```

Comment:
in real scenarios these id's would be different, right?

Reply:
yes, you're right. It would be key.

Reply:
99ec29d